src/components/jacobian_component/jacobian.sh

```diff
@@ -81,9 +82,26 @@ setup_jacobian() {
         fi
 	fi
    
-	# Update settings in geoschem_config.yml
-	sed -i -e "s|emission_perturbation: 1.0|emission_perturbation: ${PerturbValue}|g" \
-	       -e "s|state_vector_element_number: 0|state_vector_element_number: ${xUSE}|g" geoschem_config.yml
+	# Update settings in geoschem_config.yml except for the base run
+    if [ $x -ne 0 ]; then
+	    sed -i -e "s|emission_perturbation: 1.0|emission_perturbation: ${PerturbValue}|g" \
```

Comment:
It looks like the emission perturbation will be applied for any `x > 0`, including for the last four elements reserved for BC perturbations. Does it matter if the BC perturbation simulations also have an emission perturbation, or will that be ignored?